docs/sphinx_docs/Usage/index.md

```diff
@@ -85,11 +85,16 @@ Model generation is disabled by default. There are two recommended ways to enabl
     )
     I don't known
   ```
-  *Note*: In this example the model for the statement `check_sat c2` is not a
+
+  ```{admonition} Note
+
+  In this example the model for the statement `check_sat c2` is not a
   model for the statement `check_sat c1` since `check_sat` are independent in
   the native language. The same goes for `goals`.
 
-  Using the SMT-LIB language in the input file `INPUT.smt2`:
+  ```
```

Comment:
I don't think this one should be an admonition, it is more a remark that doesn't deserve specific emphasis imo; it would be fine to keep it in the main content.